### ISSUE TITLE
Bower Error

### ISSUE THREAD

**Issue by anieruddha** [nobleclem/jQuery-MultiSelect#70]
Hi,

In bower.json, added following entry.

`"jQuery-MultiSelect":"2.3.6"`

Getting Following Error :
 
```
bower jQuery-MultiSelect#2.3.6       not-cached https://github.com/nobleclem/jQuery-MultiSelect.git#2.3.6
bower jQuery-MultiSelect#2.3.6          resolve https://github.com/nobleclem/jQuery-MultiSelect.git#2.3.6
bower jQuery-MultiSelect#2.3.6     ENORESTARGET No tag found that was able to satisfy 2.3.6

Additional error details:
No versions found in https://github.com/nobleclem/jQuery-MultiSelect.git
```

**Comment by anieruddha** [comment]
I resolve this using something like 

`"jQuery-MultiSelect":"nobleclem/jQuery-MultiSelect#a396500"
`
Close the issue, if you dont want create tag.

**Comment by nobleclem** [comment]
Not sure I want to tag at this point.  Not unless this is a bigger issue for more people.  I will leave it open for feedback.  If it gets a few people wanting the project tagged I will do it.  If not then its not worth the extra step.

**Comment by li-ar** [comment]
It would be great to have an easy way to install  this great plugin via bower. I try to add jQuery-MultiSelect
dependence in my yii2 project but get the same error "The requested package could not be found in any version"

**Comment by nobleclem** [comment]
Did you try the format that @anieruddha proposed?

Currently the latest commit hash is `a4efb69` instead of `a396500` when they posted their solution.

**Comment by li-ar** [comment]
Yep, I use assets plugin for composer and it seems to me that its does not recognize record like this "nobleclem/jQuery-MultiSelect#commit-version".
